src/main/scala/redis/api/Clusters.scala

```diff
@@ -70,4 +70,26 @@ case class ClusterSlots() extends RedisCommand[MultiBulk,Seq[ClusterSlot]] {
     }
 
   }
+}
+
+case class ClusterInfo() extends RedisCommand[Bulk, Map[String, String]] {
+  val isMasterOnly = false
+  val encodedRequest: ByteString = encode("CLUSTER INFO")
+  def decodeReply(b: Bulk): Map[String, String] = {
+    b.response.map(_.utf8String.split("\r\n").map(_.split(":")).map(s => (s(0),s(1))).toMap).getOrElse(Map.empty)
+  }
+  override val decodeRedisReply: PartialFunction[ByteString, DecodeResult[Bulk]] = {
+    case s => RedisProtocolReply.decodeReplyBulk(s)
+  }
+}
+
+case class ClusterNodes() extends RedisCommand[Bulk, Array[Map[String, String]]] {
+  val isMasterOnly = false
+  val encodedRequest: ByteString = encode("CLUSTER NODES")
+  def decodeReply(b: Bulk): Array[Map[String, String]] = {
+    b.response.map(_.utf8String.split("\n").map(_.split(" ")).map(s => scala.collection.immutable.Map("id" -> s(0), "ip-port" -> s(1), "flags" -> s(2), "master" -> s(3), "ping-sent" -> s(4), "pong-recv" -> s(5), "config-epoch" -> s(6), "link-state" -> s(7), "slots" -> s.drop(8).mkString(" ")))).getOrElse(Array.empty)
```

Comment:
I have reviewed the doc at https://redis.io/commands/cluster-nodes
you should definitely use a case class (I am okay with being a bit lazy and use a case class of string (slot being a List[String]))